docs/maui/converters/string-to-list-converter.md

```diff
@@ -119,23 +119,25 @@ class StringToListConverterPage : ContentPage
 {
     public StringToListConverterPage()
     {
-		Content = new VerticalStackLayout
+        Content = new VerticalStackLayout
         {
             Children =    
             {
                 new Entry { Placeholder = "Enter some text separated by ',' or '.' or ';'" }
-                    .Bind(Entry.TextProperty, path: nameof(ViewModel.MyValue)),
+                    .Bind(
+                        Entry.TextProperty, 
+                        static(ViewModel vm => vm.MyValue)),
 
                 new CollectionView
-    			{
-    				ItemTemplate = new DataTemplate(() => new Label().Bind(Label.TextProperty, path: "."))
-    			}.Bind(CollectionView.ItemsSourceProperty,
+                {
+                    ItemTemplate = new DataTemplate(() => new Label().Bind(Label.TextProperty, path: "."))
+                }.Bind(CollectionView.ItemsSourceProperty,
```

Comment:
Let's update this binding to use TypedBindings

Reply:
We can't as we don't have the converter support yet